docs/collator-instructions.md

```diff
@@ -0,0 +1,177 @@
+# How to set up an InvArch Collator
+
+Make sure you are logged in as a user with root privileges, otherwise:
+
+``sudo su -``
+
+Create a new service user to run your collator service:
+`sudo useradd --no-create-home --shell /usr/sbin/nologin tinkernet-collator`
+
+Install some required stuff:
+
+``curl https://sh.rustup.rs -sSf | sh``
+(choose option 1 - Proceed with installation (default))
+You may need to restart your system before the next steps.
+```
+rustup install nightly-2021-11-02
+rustup target add wasm32-unknown-unknown --toolchain nightly-2021-11-02
+apt install cmake git clang libclang-dev
+```
+Type Y to proceed.
+
+Clone the repo:
+
+ ``git clone git@github.com:InvArch/InvArch-Node.git``
+
+Type the following command, and make sure you can see "gm-chain", if not then you likely did something wrong.
+
+  ``ls -la``
+
+Enter the repo and check out the most recent tagged release of code (https://github.com/GMorDIE/gm-chain/releases)
+
+```
+cd InvArch-Node
+git checkout v1.2.0
+```
+
+You need to compile the code, this will take quite a while depending on your system (30+ minutes is normal):
+ ``cargo build --release``
+
+Mid-way through compiling, you likely need to enter this command when prompted:
+
+``rustup target add wasm32-unknown-unknown``
+
+Resume compiling:
+
+``cargo build --release``
+
```

Comment:
```suggestion
```
This shouldn't be necessary because of the reason described in the first comment